### ISSUE TITLE
Properties' totalSupply is too little

### ISSUE THREAD

**Issue by aggre** [dev-protocol/protocol#332]
# Problem subject

- [x] Contracts
- [ ] Tests
- [ ] Other

# Description

A `totalSupply` of a Property contract created by PropertyFactory is `0.00000000001`.

https://github.com/dev-protocol/protocol/blob/557ff4660f5174946a0632c7dd3a77fa9d0cd60c/contracts/src/property/Property.sol#L13-L14

# Expected behavior/contents

If a value of `decimals` is 18, `totalSupply` is should be `10000000000000000000000000` to describe 10,000,000.

```solidity
uint8 private constant _decimals = 18;
uint256 private constant _supply = 10000000000000000000000000;
```

# Actual behavior/contents

https://github.com/dev-protocol/protocol/blob/557ff4660f5174946a0632c7dd3a77fa9d0cd60c/contracts/src/property/Property.sol#L13-L14

# Code of Conduct

By submitting this pull request, I confirm I've read and complied with the [CoC](https://github.com/dev-protocol/repository-token/blob/master/CODE_OF_CONDUCT.md) 🖖

**Comment by aggre** [comment]
I'm thinking the update flow like the following.

## 1. Update the PropertyFactory

Fix the incorrect Property, then re-deploy and update the PropertyFactory.

By this update, become the correct Property from a new Property.

## 2. Implement the Cancelable Authentication

Update the already authenticated assets to cancelable.

By this update, a user will be able to replace Property and Metrics or delete Metrics.

## 3. Re-deploy the NpmMarket

Re-deploy the NpmMarket, and create a new Market using the NpmMarket as Behavior. Re-run NpmMarket's migration, association each Property and npm packages.

Then, remove old Metrics and Market that no longer needed.

**Comment by aggre** [comment]
The part of this issue(step 1) is already fixed by #335 .

**Comment by aggre** [comment]
This issue is already solved. (But the step of "3. Re-deploy the NpmMarket" is don't execute by security reason)
